playwright/test/view-items.test.ts

```diff
@@ -36,11 +42,64 @@ describe('Scenario 1: A user wants a large-scale view of an item', () => {
     await multiVolumeItem();
     await page.waitForSelector(fullscreenButton);
     await page.click(fullscreenButton);
-    // check full screen
+    // TODO: e.g. test for existence of document.fullscreenElement (can't figure out how to do this with Playwright)
```

Comment:
Sorry, being - but could you explain this TODO?

Reply:
We want to check that clicking the 'fullscreen' button actually launches fullscreen (which would be true if `document.fullscreenElement` wasn't `null`)